base/applications/mscutils/servman/lang/pl-PL.rc

```diff
@@ -78,31 +78,31 @@ BEGIN
     VK_F5, ID_REFRESH, VIRTKEY
 END
 
-IDD_DLG_GENERAL DIALOGEX 6, 6, 253, 225
-CAPTION "Ogólny"
+IDD_DLG_GENERAL DIALOGEX 6, 6, 253, 232
+CAPTION "Ogólne"
 FONT 8, "MS Shell Dlg", 0, 0
 STYLE DS_SHELLFONT | WS_BORDER | WS_VISIBLE | WS_DLGFRAME | WS_SYSMENU | WS_GROUP | WS_TABSTOP
 BEGIN
-    LTEXT "", IDC_SERV_NAME, 70, 11, 176, 11, WS_CHILD | WS_VISIBLE
+    LTEXT "", IDC_SERV_NAME, 70, 11, 176, 8, WS_CHILD | WS_VISIBLE
     EDITTEXT IDC_DISP_NAME, 70, 29, 176, 13, WS_CHILD | WS_VISIBLE | WS_TABSTOP | ES_READONLY
     EDITTEXT IDC_DESCRIPTION, 70, 46, 176, 24, WS_CHILD | WS_VISIBLE | WS_VSCROLL | WS_TABSTOP | ES_MULTILINE | ES_READONLY
     EDITTEXT IDC_EXEPATH, 6, 86, 240, 13, WS_CHILD | WS_VISIBLE | WS_TABSTOP | ES_READONLY
     COMBOBOX IDC_START_TYPE, 70, 107, 176, 40, WS_CHILD | WS_VISIBLE | WS_TABSTOP | CBS_DROPDOWNLIST
-    PUSHBUTTON "Uruchom", IDC_START, 6, 155, 54, 15, WS_DISABLED
-    PUSHBUTTON "Zatrzymaj", IDC_STOP, 68, 155, 54, 15, WS_DISABLED
-    PUSHBUTTON "Wstrzymaj", IDC_PAUSE, 130, 155, 54, 15, WS_DISABLED
-    PUSHBUTTON "Wznów", IDC_RESUME, 192, 155, 54, 15, WS_DISABLED
-    LTEXT "Nazwa usługi:", IDC_STATIC, 4, 11, 53, 11
-    LTEXT "Nazwa wyświetlana:", IDC_STATIC, 4, 32, 65, 11
-    LTEXT "Opis:", IDC_STATIC, 4, 48, 53, 11
-    LTEXT "Ścieżka do aplikacji", IDC_STATIC, 6, 76, 82, 9
-    LTEXT "Typ uruchomienia:", IDC_STATIC, 6, 109, 59, 11
-    LTEXT "Stan usługi:", IDC_STATIC, 4, 138, 53, 11
-    LTEXT "", IDC_SERV_STATUS, 70, 138, 176, 11, WS_CHILD | WS_VISIBLE
-    LTEXT "Możesz określić parametry początkowe, które będą użyte przy uruchomieniu usługi z tego miejsca.", IDC_STATIC, 6, 177, 240, 15
-    LTEXT "Parametry uruchomienia:", IDC_STATIC, 6, 196, 58, 21
+    PUSHBUTTON "Uruchom", IDC_START, 6, 155, 54, 14, WS_DISABLED
+    PUSHBUTTON "Zatrzymaj", IDC_STOP, 68, 155, 54, 14, WS_DISABLED
+    PUSHBUTTON "Wstrzymaj", IDC_PAUSE, 130, 155, 54, 14, WS_DISABLED
+    PUSHBUTTON "Wznów", IDC_RESUME, 192, 155, 54, 14, WS_DISABLED
+    LTEXT "Nazwa usługi:", IDC_STATIC, 4, 11, 53, 8
+    LTEXT "Nazwa wyświetlana:", IDC_STATIC, 4, 32, 65, 8
+    LTEXT "Opis:", IDC_STATIC, 4, 48, 53, 8
+    LTEXT "Ścieżka do wykonywalnego:", IDC_STATIC, 6, 76, 91, 8
```

Comment:
The previous translation was better. Either improve the new one or revert to previous translation.

Reply:
look original english translation...

Reply:
That doesn't make much sense. The Polish word "wykonywalny" is not used to name "an executable" just by itself, it is used in conjunction with "plik" (file). If there's a problem with string length constraints, then either try to figure out a different translation or revert the previous one.